oasp4j-samples/oasp4j-sample-core/src/test/java/io/oasp/gastronomy/restaurant/salesmanagement/dataaccess/impl/dao/BillDaoTest.java

```diff
@@ -27,7 +27,6 @@
  * @author mvielsac
  */
 @RunWith(SpringJUnit4ClassRunner.class)
-@TestExecutionListeners({ TransactionalTestExecutionListener.class, DependencyInjectionTestExecutionListener.class })
```

Comment:
See https://github.com/oasp/oasp4j/issues/174#issuecomment-130468017 for explanation